xrdsst/controllers/diagnostics.py

```diff
@@ -0,0 +1,207 @@
+from cement import ex
```

Comment:
I think this whole code could be refactor, because it's practically the same for all diagnostics, look what I did for example in the cert controller with the class cert operation, you could have more or less the same, one object with the API method and other call parse_response or print_response or something like that.

Reply:
Refactored the code and removed 3 similar methods